tools/swift-ide-test/swift-ide-test.cpp

```diff
@@ -1185,6 +1191,12 @@ static int doBatchCodeCompletion(const CompilerInvocation &InitInvok,
       continue;
     }
 
+    bool failureExpected = false;
+    if (!Token.Xfail.empty()) {
+      failureExpected = true;
+      llvm::errs() << "Xfail: " << Token.Xfail << "\n";
+    }
+
```

Comment:
Doesn't really matter but you could make this:
```
bool failureExpected = !Token.Xfail.empty();
if (failureExpected) {
  llvm::errs() << ...;
}
```

Reply:
👍 Let's save one line.